src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.spec.ts

```diff
@@ -169,14 +165,10 @@ describe('OnlineGameWrapperComponent Lifecycle', () => {
 
             testUtils.detectChanges();
 
-            const partCreationId: DebugElement = testUtils.findElement('#partCreation');
-            const gameId: DebugElement = testUtils.findElement('#game');
-            const p4Tag: DebugElement = testUtils.findElement('app-p4');
-
             expect(wrapper.gameStarted).withContext('game should be started').toBeTrue();
-            expect(partCreationId).withContext('partCreation id should be absent after startGame call').toBeFalsy();
```

Comment:
P-e remettre le contenu de ces erreurs qqpart dans le test as commentaires ? à priori enlever du contexte n'est pas souhaitable

Reply:
à partir du moment où on a une test spécifique "l'élément doit exister" plutôt que "expect to be truthy", le contexte on l'a déjà pas mal depuis notre test spécifique, ce qui reste c'est du surplus de la poubelle.
Je GWTise ça correctement plutôt que de juste copier de la veille poubelle en commentaire !